src/Form/index.tsx

```diff
@@ -922,6 +924,7 @@ function Form({
       .fetchForm(initialValues, language)
       .then(async (data: any) => {
         await updateCustomHead(data.custom_head ?? '');
+        setFormName(data.form_name); // API definition guarantees this.
```

Comment:
let's move this to the next .then function, this one is special for the custom_head sinc eit needs to occur before the other stuff

Reply:
done